pkg/sources/git/git.go

```diff
@@ -1005,15 +1005,13 @@ func (s *Git) handleBinary(ctx context.Context, gitDir string, reporter sources.
 		return nil
 	}
 
-	var handlerOpts []handlers.Option
 	if s.skipBinaries {
-		handlerOpts = append(handlerOpts, handlers.WithSkipBinaries(true))
-		if common.IsBinary(path) {
-			fileCtx.Logger().V(5).Info("skipping binary file")
-			return nil
-		}
+		fileCtx.Logger().V(5).Info("skipping binary file", "path", path)
+		return nil
 	}
 
+	var handlerOpts []handlers.Option
```

Comment:
__optional nitpick__: Probably best to delete this and it's only usage on line 1052/1050, just so a future reader won't think we're actually using handlerOpts at the moment.